### PR TITLE
Feature/home-page

### PR DESCRIPTION
**Pull request by Abdalrazaq94**
## Feature/home-page

# I have Created a home page for the first component. (Home body)

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Review by shieldo — APPROVED** [review]
There's going to need to be some work to make the site work well on e.g. tablets and mobiles, but in terms of making the image appear on the top area of the homepage and hooking up the links correctly, this is a great start!

File: client/src/pages/component/homeBody/Homebody.css [hunk 1]
```diff
@@ -1,4 +1,88 @@
-.homebody{
-    text-align: center;
-    margin-top: 100px;
+.homebody-container {
+	background-image: url(../image/home-body.jpg);
+	background-repeat: no-repeat;
+	width: 100%;
+	height: 650px;
+	background-size: contain;
```
Comment (zhkvivan) [inline client/src/pages/component/homeBody/Homebody.css:6]: Screenshot 2022-11-18 at 14 57 32

try backgroudn size cover.

File: client/src/pages/component/homeBody/Homebody.css [hunk 2]
```diff
@@ -1,4 +1,88 @@
...
+	background-position: cover;
+}
+.body-main-container {
+	position: relative;
+	height: 100vh;
+}
```
Comment (zhkvivan) [inline client/src/pages/component/homeBody/Homebody.css]: are these realy needed? I turned them off and nothing changed
Screenshot 2022-11-18 at 14 59 56

Comment (Abdalrazaq94) [reply client/src/pages/component/homeBody/Homebody.css]: deleted, done

File: client/src/pages/component/homeBody/Homebody.css (hunk 1 above)
Comment (Abdalrazaq94) [reply client/src/pages/component/homeBody/Homebody.css:6]: done

File: client/src/pages/component/homeBody/Homebody.css (hunk 2 above)
Comment (Abdalrazaq94) [reply client/src/pages/component/homeBody/Homebody.css]: @zhkvivan should i merge it

File: client/src/pages/component/homeBody/Homebody.css (hunk 1 above)
Comment (zhkvivan) [reply client/src/pages/component/homeBody/Homebody.css:6]: Cool but do you think `background-size: contain;` does the job better than `background-size: cover;`?

File: client/src/pages/component/homeBody/Homebody.css [hunk 3]
```diff
@@ -1,4 +1,84 @@
...
+	margin: auto;
+	height: 60vh;
+	text-align: center;
+	align-items: center;
+	justify-content: center;
+	position: absolute;
+	right: 0;
+	left: 0;
+	top: 0;
+	bottom: 0;
```
Comment (zhkvivan) [inline client/src/pages/component/homeBody/Homebody.css]: Here you have a few properties from flex layout (but you missed `display: flex`) and a few properties from absolute positioning. I would suggest you to decide wich way you want to position your elements  - flexbox or absolute positioning and stick to it

**Pull request merged**